modules/eks/cluster/remote-state.tf

```diff
@@ -48,7 +48,9 @@ module "eks" {
     eks_managed_node_workers_role_arns = []
     fargate_profile_role_arns          = []
     fargate_profile_role_names         = []
-    eks_cluster_identity_oidc_issuer   = ""
+    # eks_cluster_identity_oidc_issuer will never exist on the first deployment because the component itself creates the OIDC provider,
+    # yet the value must not be empty when passed to the aws-ebs-csi-driver and vpc-cni addons
+    eks_cluster_identity_oidc_issuer = "https://placeholder.example.com/id/ABCDEF00000000000000000000000000"
```

Comment:
This would not fix the problem, it would only hide it. The add-on will still not have the OIDC issuer it needs, but with this it will accept the bad value and defer the error to logs after it is deployed.